geometry_msgs/Vector3 start_pose
geometry_msgs/Vector3 goal_pose
---
geometry_msgs/Vector3[] trajectory


